hail/python/hailtop/aiocloud/aiogoogle/session.py

```diff
@@ -6,7 +6,11 @@
 
 
 class GoogleSession(BaseSession):
-    def __init__(self, *, credentials: Optional[GoogleCredentials] = None, params: Optional[Mapping[str, str]] = None, **kwargs):
+    def __init__(self, *, credentials: Optional[GoogleCredentials] = None, credentials_file: Optional[str] = None,
+                 params: Optional[Mapping[str, str]] = None, **kwargs):
         if credentials is None:
-            credentials = GoogleCredentials.default_credentials()
+            if credentials_file:
+                credentials = GoogleCredentials.from_file(credentials_file)
+            else:
+                credentials = GoogleCredentials.default_credentials()
```

Comment:
if credentials is not None, assert credentials_file is None.

Reply:
I want it to fallback on the default credentials instead.

Reply:
What I meant is that it's a bug to pass both credentials and credentials_file to GoogleSession (or at least, it should be a bug).

e.g.
```python3
assert credentials is None or credentials_file is None, f'specify only one of credentials or credentials_file: {(credentials, credentials_file)}'
```